app/src/main/kotlin/com/simplemobiletools/smsmessenger/activities/ThreadActivity.kt

```diff
@@ -114,6 +112,9 @@ class ThreadActivity : SimpleActivity() {
 
     private var isAttachmentPickerVisible = false
 
+    private val binding by viewBinding(ActivityThreadBinding::inflate)
+    private lateinit var messageHolderBinding: LayoutThreadSendMessageHolderBinding
```

Comment:
why is it a lateinit and not initiated consistently like everything else?

Reply:
`binding.threadSendMessageHolder` is used in like 40 places so I used `messageHolderBinding` variable to keep it short. Renaming the view id instead...